packages/preview-middleware-client/src/adp/controllers/ShowFileExistDialog.controller.ts

```diff
@@ -0,0 +1,81 @@
+import JSONModel from 'sap/ui/model/json/JSONModel';
+import type Dialog from 'sap/m/Dialog';
+
+import type Event from 'sap/ui/base/Event';
+
+import type SimpleForm from 'sap/ui/layout/form/SimpleForm';
+
+export interface ShowFileExist {
```

Comment:
Interface name sounds like a command. Maybe better to name it like FileExistsDialogOptions